client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx

```diff
@@ -646,22 +649,37 @@ export default function CampaignItemDetails( props: Props ) {
 						<div className="campaign-item-details__main-stats-container">
 							<div className="campaign-item-details__secondary-stats">
 								<div className="campaign-item-details__secondary-stats-row">
-									<div>
-										<span className="campaign-item-details__label">{ __( 'Weekly budget' ) }</span>
-										<span className="campaign-item-details__text wp-brand-font">
-											{ ! isLoading ? weeklyBudgetFormatted : <FlexibleSkeleton /> }
-										</span>
-										<span className="campaign-item-details__details">
-											{ ! isLoading ? (
-												/* translators: Daily average spend. dailyAverageSpending is the budget */
-												translate( 'Daily av. spend: $%(dailyAverageSpending)s', {
-													args: { dailyAverageSpending: dailyAverageSpending },
-												} )
-											) : (
-												<FlexibleSkeleton />
-											) }
-										</span>
-									</div>
+									{ isLessThanOneWeek ? (
+										<div>
+											<span className="campaign-item-details__label">{ __( 'Daily budget' ) }</span>
```

Comment:
i think we need a isLoading condition here and display the skeleton. 
as the page loads (I have a weekly campaign) I see the Daily budget when the page loads.. then it switches to weekly

Reply:
https://github.com/user-attachments/assets/101f147d-0cfb-4130-9af1-ff1d27505378